vault/expiration.go

```diff
@@ -2418,6 +2415,140 @@ func (m *ExpirationManager) markLeaseIrrevocable(ctx context.Context, le *leaseE
 	m.nonexpiring.Delete(le.LeaseID)
 }
 
+func (m *ExpirationManager) getNamespaceFromLeaseID(ctx context.Context, leaseID string) (*namespace.Namespace, error) {
+	_, nsID := namespace.SplitIDFromString(leaseID)
+	leaseNS := namespace.RootNamespace
+	var err error
```

Comment:
this looks like an unneeded declaration, but it avoids creating a `leaseNS` with scope inside of the if (and not actually setting the one we want)